pkg/cli/root_test.go

```diff
@@ -16,13 +16,13 @@ func TestUsage(t *testing.T) {
 
 	Convey("Test usage", t, func(c C) {
 		os.Args = []string{"cli_test", "help"}
-		err := cli.NewRootCmd().Execute()
+		err := cli.NewRootCmd("/tmp").Execute()
```

Comment:
Use ioutil.Tempdir() instead of "/tmp" explicitly, there may not be a /tmp on Windows where one is compiling this code, let golang runtime decide that location.